README_el.md

```diff
@@ -55,40 +54,40 @@ experiment matrixはカンマ区切りで（csv形式）。
 |  Treg_LN_1  | hoge/SRR5385247 | SE | Treg | ...|
 |  Treg_LN_2  |  hoge/SRR5385248  | SE  | Treg | ... |
 
-- nameはアンダーバー区切りでcondition、replicateをつなげて書く。
-- 前3列は必須。
-- 自前のfastq fileを使いたいときは`--fastq`をつける。拡張子は`.fq`, `.fq.gz`, `.fastq`, `fastq.gz`のみに対応。
-- fastq fileは`fastq.gz`もしくは`_1.fastq.gz`,`_2.fastq.gz`を除いたpathを。例えば`hoge/SRR5385247.fastq.gz`なら`hoge/SRR5385247`と記載。
-- suffixが`_1.fastq.gz`,`_2.fastq.gz`ではない場合は-s1, -s2オプションをつける。
-- `../fq/**.fastq.gz`など、実行ディレクトリより上の階層を指定することはdockerの都合上不可能だが、symbolic linkを貼ることで回避できる。
+- name is written by connecting condition and replicate with underscores separated.
+- The first three columns are required.
+- If you want to use your own fastq file, add `--fastq`.The extension supports only `.fq`,` .fq.gz`, `.fastq` and` fastq.gz`.
```

Comment:
If you want to use your own fastq files, add `--fastq`. The extension supports only `.fq`,` .fq.gz`, `.fastq` and` fastq.gz`.